pallets/pool-system/src/impls.rs

```diff
@@ -84,21 +84,9 @@ impl<T: Config> TrancheTokenPrice<T::AccountId, T::CurrencyId> for Pallet<T> {
 			.calculate_prices::<T::BalanceRatio, T::Tokens, _>(total_assets, now)
 			.ok()?;
 
-		let base = pool
-			.tranches
-			.tranche_currency(TrancheLoc::Id(tranche_id))?
-			.into();
-
 		let price = prices.get(tranche_index).cloned()?;
 
-		Some(PriceValue {
```

Comment:
@hieronx you added this change. Why did you wanted to have the `TrancheCurrency` and the `PoolCurrency` named here?

Reply:
I don't fully remember. I think we just set up the `PriceValue` for some other purpose, and then used that here.